client/src/Ratings.jsx

```diff
@@ -52,6 +56,54 @@ class Ratings extends Component {
   // look into retis
   // elastic search db
   //
+
+  onAddReview(obj) {
+    console.log('In the ratings component')
+    console.log(obj);
+    // Create post request to send back data.
+  }
+
+  onUpdateReview(id, action){
+    axios.put(`/api/reviews/${id}/${action}`)
+    .catch((err) => {
+      console.log('Error in updating the review action', err);
```

Comment:
Instead of console logging errors, we should return them so the user is presented with an error detailing what happened